src/main/java/com/feedbacksolution/model/Tweet.java

```diff
@@ -0,0 +1,211 @@
+package com.feedbacksolution.model;
```

Comment:
Do we need all of this in our object. I think we should keep only those that are useful to us